Dockerfile

```
@@ -16,5 +16,9 @@ RUN apk add --no-cache \
 ENTRYPOINT ["./docker-entrypoint.sh"]
 WORKDIR /worker
 
+RUN addgroup -g 1000 -S bp3global && \
+    adduser -u 1000 -S bp3user -G bp3global -h /worker -s /bin/bash -D bp3global
```

```suggestion
    adduser -u 1000 -S bp3user -G bp3global -h /worker -s /bin/bash -D bp3user
```
The last arg is supposed to be the _LOGIN_

---

Suggested changes all agreed.